testutils/darknode.go

```diff
@@ -63,9 +63,7 @@ func (node *MockDarknode) Run(done <-chan struct{}) {
 		server.Close()
 	}()
 
-	if err := server.ListenAndServe(); err != nil {
-		log.Printf("failed to serve: %v", err)
-	}
+	server.ListenAndServe()
```

Comment:
Reason why the error check was removed?

Reply:
The `ListenAndServe` will always return a non-nil err and I think it's ok to ignore the error.